ffc-ahwr-event/index.js

```diff
@@ -9,6 +10,7 @@ module.exports = async function (context, message) {
   if (event.name === 'send-session-event') {
     if (validateEvent(event)) {
       await saveEvent(context, event)
+      await sendEvent(context, event)
```

Comment:
Think this needs renaming to make it more relevant to sending an event to the SOC protective monitoring